packages/icon-button/icon-button.scss

```diff
@@ -0,0 +1,12 @@
+@import "~@fremtind/jkl-core/mixins/_helpers.scss";
+
+.jkl-icon-button {
+    @include reset-outline;
+    background-color: transparent;
+    cursor: pointer;
```

Comment:
We should probably also add `color: inherit`, since most browsers specify `color` on buttons explicitly.

Reply:
done: c03900e9673d01a59242f74c77eee7a5da991490